tools/convert-to-centos-stream.yml

```diff
@@ -8,6 +8,39 @@
   gather_facts: true
   tasks:
 
+  - name: List repo files
+    find:
+      paths: /etc/yum.repos.d/
+      file_type: file
+      patterns: 'CentOS-Linux-*.repo'
+    register: pre_stream_repo_files
+    when: ansible_distribution == 'CentOS'
+
+  # From ansible docs: 'replace: If not set, matches are removed entirely.'
+  - name: Remove all mirrorlists
+    replace:
+      path: "{{ item.path }}"
+      regexp: '^mirrorlist=.*'
+    with_items: "{{ pre_stream_repo_files.files }}"
+    when: ansible_distribution == 'CentOS'
+
+  - name: Uncomment baseurls
+    replace:
+      path: "{{ item.path }}"
+      regexp: '^mirrorlist=.*'
+      regexp: '^\s*#*\s*(baseurl=.*)'
+      replace: '\1'
+    with_items: "{{ pre_stream_repo_files.files }}"
+    when: ansible_distribution == 'CentOS'
+
+  - name: Point baseurls to archive server
+    replace:
+      path: "{{ item.path }}"
+      regexp: 'mirror.centos.org/\$contentdir/\$releasever'
+      replace: 'vault.centos.org/8.5.2111'
```

Comment:
This feels like something that could go stale without us really noticing; should it maybe be abstracted to a variable?

Reply:
Eh.  My thinking was

1. vault.centos.org is supposed to be the vault.  Like it doesn't change.
2. There will never be a newer version of CentOS 8.  8.5.2111 is the last one.

Reply:
Oh, of course! 🤦